server/index.js

```diff
@@ -28,7 +28,7 @@ app.use(cors())
 // app.use('/api/todos', todoRouter)
 app.get('/api/todos', handleList)
 app.post('/api/todos', handleCreate)
-app.get('/api/todos/:id', handleSingleTodo)
+app.get('/api/todos/:_id', handleSingleTodo)
 app.put('/api/todos/:id', handleUpdate)
 app.delete('/api/todos/:id', handleDelete)
```

Comment:
On some routes, the `params` name is `id` others `_id`. This only dictates the name of the variable on `params`, so consider picking one naming convention and sticking with it for all of your routes.